client/src/hooks/pages copy/index.ts

```diff
@@ -0,0 +1,12 @@
+import { useGetPagesId } from '@/types/generated/page';
```

Comment:
Is the path of this file correct?

Reply:
It is, why?

Reply:
The path of the file is `pages copy/index.ts`. It seems like “copy” is a mistake, no?